js homework week2/file1.js

```diff
@@ -0,0 +1,7 @@
+let turkish = 'Merhaba Dunya!';
+let english = 'Hello World!';
+let dutch = 'Hallo Werld!';
+
+console.log(turkish);
+console.log(english);
+console.log(dutch);

```

Comment:
Nicenice!

I think you forgot an 'e' though for the dutch one:

"Hallo Wereld!" :)

Reply:
jaa. het is wereld! done